### DISCUSSION TITLE
Will future versions of `murex-linux-amd64` be compatible with not so recent version of `libc`?

### DISCUSSION THREAD

**Discussion by nicck** [lmorg/murex#755] [Q&A] [answered]
Ubuntu 20.04 LTS is still around, but unfortunately it's not trivial to run `murex-linux-amd64` on it b/c of the following issue:
```
./murex-linux-amd64: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by ./murex-linux-amd64)
./murex-linux-amd64: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by ./murex-linux-amd64)
./murex-linux-amd64: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by ./murex-linux-amd64)
```
Is there an easy way to make it work, aside from rebuilding murex-linux-amd64 myself or upgrading Ubuntu?

**Comment by lmorg** [comment]
Hi @nicck 

I believe the only requirement on libc is sqlite3. Fortunately it's also pretty trivial either without needing sqlite3, or to use an sqlite library that doesn't depend upon libc. So I'll take a look at this later today and see if I can re-run the build pipeline to generate binaries that aren't linked against glibc.

**Reply by lmorg** [reply]
~~@nicck are you able to re-download `murex-linux-amd64` please? The build went through successfully last night.~~

Scratch that, there's a couple more references to libc that I need to fix

**Reply by lmorg** [reply, marked as answer]
@nicck sorry for the false start there earlier. The downloads should be libc free now (I've tested on a Ubuntu 20.04 machine and the `murex-linux-amd64` build worked)

**Reply by nicck** [reply]
It works! Thank you for the quick reply and resolution!
